### PR TITLE
add Alpine.39 and update ARM64 container

### PR DESCRIPTION
**Pull request by wfurt**
uses new container infrastructure + friendly names.

### CODE REVIEW

File: eng/pipelines/linux.yml [hunk 1]
```diff
@@ -93,13 +93,13 @@ jobs:
 
         - ${{ if eq(parameters.isOfficialBuild, 'false') }}:
           - linuxDefaultQueues: Centos.7.Amd64.Open+RedHat.7.Amd64.Open+Debian.8.Amd64.Open+Ubuntu.1604.Amd64.Open+Ubuntu.1804.Amd64.Open+OpenSuse.42.Amd64.Open+Fedora.27.Amd64.Open
-          - linuxArm64Queues: Ubuntu.1604.Arm64.Open
+          - linuxArm64Queues: '\(Ubuntu.1604.Arm64\)Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8-b049512-20190321153539'
```
Comment (safern) [inline eng/pipelines/linux.yml]: I'm surprised we need to escape `(` and `)` when using ' in the whole thing. Did you try without it?

Comment (wfurt) [reply eng/pipelines/linux.yml]: I did it in original PR for arm64. We pass that string to shell script and that blows up with "invalid expression" error. 
I can investigate more as that should not be needed IMHO to do helix run.

Comment (MattGal) [reply eng/pipelines/linux.yml]: have you tried removing the quotes here?, i.e. 
```
- linuxArm64Queues: (Ubuntu.1604.Arm64)Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8-b049512-20190321153539
```
???

Comment (wfurt) [reply eng/pipelines/linux.yml]: for quotes, I don't remember as I did few variations. I can give it try. What ever value we have, we pass in to build.sh and that seems to write it to another file for Helix execution.

Comment (safern) [reply eng/pipelines/linux.yml]: we pass it to `msbuild.sh sendtohelix.proj` and then that one passes it down to Helix.

**Pull request merged**